spec/unit/puppet/type/windowsfeature/windowsfeature_spec.rb

```diff
@@ -3,16 +3,18 @@
 require 'spec_helper'
 
 describe Puppet::Type.type(:windowsfeature) do
-  let :provider_class do
-    described_class.provide(:fake) { mk_resource_methods }
-  end
-  let(:provider) { provider_class.new }
   before do
-    described_class.stubs(:defaultprovider).returns provider_class
-    described_class.stubs(:provider).returns provider
+    @class = described_class
+    @provider_class = @class.provide(:fake) { mk_resource_methods }
```

Comment:
so you mean you get spec failures with the code from master? But travis was green on the last merge? In which env did that happen?

---

The preffered way seems to be to not use instance variables, that's why I changed it.